ethcore/src/client/test_client.rs

```diff
@@ -521,6 +524,66 @@ impl BlockInfo for TestBlockChainClient {
 	}
 }
 
+// Basing test implementations off/stealing them
+// from `ethcore/src/verification/verification.rs`
```

Comment:
Can't we implement those methods near `TestBlockChainClient` and share them for both/all tests?